timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -24,12 +24,16 @@ limitations under the License.
           <v-img src="/dist/timesketch-color.png" max-height="25" max-width="25" contain></v-img>
         </router-link>
       </v-avatar>
-      <span v-if="!hasTimelines" style="font-size: 1.1em">{{ sketch.name }} </span>
+      <span v-show="!showLeftPanel && !loadingSketch" class="mr-1" style="font-size: 1.1em">{{ sketch.name }} </span>
 
       <v-btn icon v-show="!showLeftPanel && !loadingSketch" @click="toggleLeftPanel" class="ml-n1">
         <v-icon>mdi-menu</v-icon>
       </v-btn>
 
+      <v-btn v-show="currentRouteName !== 'Explore'" :to="{ name: 'Explore', params: { sketchId: sketchId } }" color="primary" small text class="ml-3">
+        <v-icon small left>mdi-step-backward</v-icon>
```

Comment:
can we change this to mdi-arrow-left instead?

Reply:
Sure, works as well. Done